test/compile/mark/area.test.ts

```diff
@@ -49,4 +49,38 @@ describe('Mark: Area', function() {
       assert.deepEqual(props.fill, {scale: COLOR, field: 'Miles_per_Gallon'});
     });
   });
+  
```

Comment:
If you want to punt for after presentation, at least we confirm that you have test with the examples and horizontal/vertical stacked/non-stacked area chart are still doing alright.